lib/std/os.zig

```diff
@@ -4843,6 +4869,28 @@ pub fn lseek_CUR_get(fd: fd_t) SeekError!u64 {
     }
 }
 
+const EnableInheritanceError = FcntlError || windows.SetHandleInformationError;
```

Comment:
This is missing the portable probing info if inheritance is enabled, which sounds relevant for developer.